README.md

```diff
@@ -2,30 +2,29 @@
 
 # Verrazzano Operator
 
-The Verrazzano Operator is the Kubernetes operator that runs in the Verrazzano Management Cluster,
-watches local custom resource definitions for models/bindings, and launches micro operators
-into Verrazzano Managed Clusters.
+The Verrazzano Operator is the Kubernetes operator that runs in the Verrazzano management cluster,
+watches local custom resource definitions for models and bindings, and launches micro-operators
+into Verrazzano managed clusters.
 
-## Project Status
+## Status
 
 Current functionality:
-- Learns about Managed Clusters through a `VerrazzanoManagedCluster` custom resource, and then creates
+- Learns about managed clusters through a `VerrazzanoManagedCluster` custom resource, and then creates
   Kubernetes clients and watchers for those clusters.
 - Watches the local Kubernetes cluster in the default namespace for `VerrazzanoModel` and `VerrazzanoBinding` custom resources.
-- A `VerrazzanoModel`/`VerrazzanoBinding` pair describes a Verrazzano application and defines which application components should be installed on which VerrazzanoManagedCluster.
-- On creation or update of a `VerrazzanoModel`/`VerrazzanoBinding` pair, the Verrazzano Operator launches:
-  - A Verrazzano Monitoring instance.
+- A `VerrazzanoModel`/`VerrazzanoBinding` pair describes a Verrazzano application and defines which application components should be installed on which `VerrazzanoManagedCluster`.
+- Upon creation or update of a `VerrazzanoModel`/`VerrazzanoBinding` pair, the Verrazzano Operator launches:
+  - A Verrazzano monitoring instance.
   - Custom resources needed to deploy application components.
-  - Agents forwarding logs and metrics to the Verrazzano Monitoring Instance.
+  - Agents forwarding logs and metrics to the Verrazzano monitoring instance.
 
-- On deletion of a `VerrazzanoBinding`, deletes all related resources deployed.
-- On updates to the Managed Cluster's objects (deployments, micro operator custom resources)
-  created by the Verrazzano Operator, triggers a reprocessing of the owning `VerrazzanoBinding`.
-  This causes any drift in expecting state to immediately be fixed by the Verrazzano Operator.
+- Upon deletion of a `VerrazzanoBinding`, deletes all deployed related resources.
+- Upon updates to the managed cluster's objects created by the Verrazzano Operator (deployments, micro-operator custom resources), triggers a reprocessing of the owning `VerrazzanoBinding`.
+  This causes any drift in expecting state to be fixed immediately by the Verrazzano Operator.
 
 ## Artifacts
 
-On a successful release (which occurs on a Git tag), this repo publishes a Docker image: `container-registry.oracle.com/verrazzano/verrazzano-operator:tag`
+Upon a successful release (which occurs on a Git tag), this repo publishes a Docker image: `container-registry.oracle.com/verrazzano/verrazzano-operator:tag`
```

Comment:
The image is now `ghcr.io/verrazzano/verrazzano-operator:tag`

Reply:
Got it, thanks.